tools/toolchain/network/docker-compose/docker-compose-base.yaml

```diff
@@ -109,7 +109,7 @@ services:
     environment:
       - GOPATH=/opt/gopath
       - CORE_VM_ENDPOINT=unix:///host/var/run/docker.sock
-      - CORE_CHAINCODE_KEEPALIVE=10
+      - CORE_CHAINCODE_KEEPALIVE=120
```

Comment:
this is non-default isn't it? 
I read a message from Dave suggesting we want to use the defaults in tests?